client/oci_env/utils.py

```diff
@@ -383,9 +383,9 @@ def exec(self, args, service=None, interactive=False, pipe_output=False):
         # docker fails on systems with no interactive CLI. This tells docker
         # to use a pseudo terminal when no CLI is available.
         if os.getenv("COMPOSE_INTERACTIVE_NO_CLI", "0") == "1":
-            cmd = [binary, "exec", self.container_name(service)] + args
+            cmd = [binary, "exec", "--privileged", self.container_name(service)] + args
```

Comment:
I'd prefer to find a way without this.

https://docs.podman.io/en/latest/markdown/podman-exec.1.html#privileged

Reply:
`binary` can also be a docker client. Not sure if it is worth doing any complicated things.

ref: https://www.redhat.com/sysadmin/podman-inside-container

Reply:
What if we add a `--privileged` flag to oci-env where you explicitly choose to run the container like that?

Reply:
Fair, any objections, @mdellweg?

Reply:
Sounds ok. Just don't break security by default for convenience.